felles/src/main/kotlin/no/nav/familie/kontrakter/felles/Ressurs.kt

```diff
@@ -29,49 +29,48 @@ data class Ressurs<T>(
         FUNKSJONELL_FEIL
     }
 
+    init {
+        if (status == Status.SUKSESS)
```

Comment:
Jeg tror jeg er helt enig med denne endring. Finnes det en risiko att dette er breaking change for noen?
Synes ju att det ikke burde være det..

Reply:
Vil brekke opprettSkyggesak, men den kan vel enkelt endres til å returnere "OK" eller "" hvis requiredNonNull står

Reply:
Tror dette er breaking for `ef-sak` - vi har et par steder hvor vi returnerer en verdi som _kan_ være null ut til frontend iallefall. Null i de tilfellene er det samme som "verdi finnes ikke ennå - og det er helt greit".

Reply:
et søk i koden på `ressurs<.*\?` gir iallefall fire treff i ef-sak. Usikker på hvordan vi løser det best - det er jo ikke "feil" at vi ikke har mellomlagret brev for en behandling... 🤔

Reply:
Det tenkte jeg ikke på. Ble bare så lei av å se tester på om data er null.

Reply:
Ja, er jo ikke uenig i dét - men så er jeg usikker på om vi burde innført noe "SUKSESS_UTEN_DATA", eller om det blir over-engineering...?